### DISCUSSION TITLE
Histoire pronunciation

### DISCUSSION THREAD

**Discussion by hugoattal** [histoire-dev/histoire#45] [Ideas]
I found these tweets pretty funny 😂 : https://twitter.com/jagoncalves14/status/1504074138629115905

But there's actually a real problem: people not knowing how to pronounce it (it might add difficulties to share it or talk about it).
I think we should add pronunciation on the website like one of these:
- \is.twaʁ\
- "Is-twa-r"

wdyt?

**Comment by FanchGadjo** [comment]
Thank you for noticing my talent on twitter ! 😆
Personally, I have never properly learnt the phonetic notation. Why not both like `vite` ?
> It's pronounced /vit/ like Veet!

is·twar or is-twar seems fine.

https://www.google.com/search?q=how+to+pronounce+histoire+in+french

**Comment by Akryum** [comment]
I added an introduction on the website which includes a note about the pronunciation: https://histoire.dev/guide/#what-is-histoire
